src/orchard/xref.clj

```diff
@@ -60,9 +65,12 @@
     (let [f-class-name (-> v .getClass .getName)
           ;; this uses the implementation detail that the clojure compiler always
           ;; prefixes names of lambdas with the name of its surrounding function class
-          deps (into #{} (comp (filter (fn [[k _v]] (clojure.string/includes? k f-class-name)))
```

Comment:
The whitespace changes add some noise. I prefer to those in separate commits, so it's clear what's the meaningful part of the change and what is simply stylistic.

Reply:
Done. Will keep in mind for next times.